docs/advanced-usage.rst

```diff
@@ -216,89 +334,197 @@ If we try to create another transaction with the same input as before, the
-..     # todo: Needs https://github.com/bigchaindb/bigchaindb-driver/issues/111
 
+and they agree that ``alice`` will be the one issuing the asset. To create a
+new digital asset with `multiple` owners, one can simply provide a
+list of ``owners_after``:
+
+.. code-block:: python
+
+    car_creation_tx = bdb.transactions.prepare(
+        operation='CREATE',
+        owners_before=alice.verifying_key,
+        owners_after=[alice.verifying_key, bob.verifying_key],
+        asset=car_asset,
+    )
+
+    signed_car_creation_tx = bdb.transactions.fulfill(
+        car_creation_tx,
+        private_keys=alice.signing_key,
+    )
+
+    sent_car_tx = bdb.transactions.send(signed_car_creation_tx
+
+One day, ``alice`` and ``bob``, having figured out how to teleport themselves,
+and realizing they no longer need their car, wish to transfer the ownership of
+their car over to ``carol``:
+
+.. code-block:: python
+
+    carol = generate_keypair()
+
+In order to prepare the transfer transaction, ``alice`` and ``bob`` need the
+input:
+
+.. code-block:: python
+
+    cid = 0
+    condition = signed_car_creation_tx['transaction']['conditions'][cid]
+    input_ = {
+        'fulfillment': condition['condition']['details'],
+        'input': {
+            'cid': cid,
+            'txid': signed_car_creation_tx['id'],
+        },
+        'owners_before': condition['owners_after'],
+    }
+
+Let's take a moment to contemplate what this ``input_`` is:
+
+.. code-block:: python
+
+    >>> input_
+    {'fulfillment': {'bitmask': 41,
+      'subfulfillments': [{'bitmask': 32,
+        'public_key': '3Cxh1eKZk3Wp9KGBWFS7iVde465UvqUKnEqTg2MW4wNf',
+        'signature': None,
+        'type': 'fulfillment',
+        'type_id': 4,
+        'weight': 1},
+       {'bitmask': 32,
+        'public_key': 'EcRawy3Y22eAUSS94vLF8BVJi62wbqbD9iSUSUNU9wAA',
+        'signature': None,
+        'type': 'fulfillment',
+        'type_id': 4,
+        'weight': 1}],
+      'threshold': 2,
+      'type': 'fulfillment',
+      'type_id': 2},
+     'input': {'cid': 0,
+      'txid': '3128572d430f9ae23010f11e444d1d12e35f90095cf5f0da98acfcada5eed0fe'},
+     'owners_before': ['3Cxh1eKZk3Wp9KGBWFS7iVde465UvqUKnEqTg2MW4wNf',
+      'EcRawy3Y22eAUSS94vLF8BVJi62wbqbD9iSUSUNU9wAA']}
+
+and the asset:
+
+.. code-block:: python
+
+    asset = signed_car_creation_tx['transaction']['asset']
+
+then ``alice`` can prepare the transfer:
+
+.. code-block:: python
+
+    car_transfer_tx = bdb.transactions.prepare(
+        operation='TRANSFER',
+        owners_after=carol.verifying_key,
+        asset=asset,
+        inputs=input_,
+    )
+
+The asset can be transfered as soon as each of the ``owners_after`` fulfills
+the transaction, that is ``alice`` and ``bob``.
+
+To do so, simply provide a list of all private keys to the fulfill method:
+
+.. code-block:: python
+
+    signed_car_transfer_tx = bdb.transactions.fulfill(
+        car_transfer_tx, private_keys=[alice.signing_key, bob.signing_key]
```

Comment:
This is a workaround for now. In real life you won't have two `private_keys` of two different users loaded in your memory. This use case will be resolved by having Alice signing and passing the transaction to Bob, so he can sign it as well, but this will happen *off–chain*.

I think this example is misleading, I'd be more explicit saying that this will change once we have a better system to have a partial transaction that can be passed around and signed.